tools/pkg/engine_build_configs/bin/run.dart

```diff
@@ -0,0 +1,141 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io' as io;
+
+import 'package:engine_build_configs/engine_build_configs.dart';
+import 'package:engine_repo_tools/engine_repo_tools.dart';
+import 'package:path/path.dart' as p;
+import 'package:platform/platform.dart';
+import 'package:process_runner/process_runner.dart';
+
+// This is an example of how to use the APIs of this library to parse and
+// execute the build configurations json files under ci/builders.
+//
+// Usage:
+// $ dart bin/run.dart [build config name] [build name]
+// For example:
+// $ dart bin/run.dart mac_unopt host_debug_unopt
+//
+// The build config names are the namess of the json files under ci/builders
+// The build names are the "name" fields of the maps in the list of "builds".
```

Comment:
This is good info, could we also add it to a README.md in `//flutter/tools/pkg/engine_build_configs/` for improved readability?

Reply:
I don't feel strongly about this, feel free to skip

Reply:
Good idea. Added a light README.md.